### DISCUSSION TITLE
The images go beyond the boundary of the article

### DISCUSSION THREAD

**Discussion by neotan** [thuliteio/doks#380] [Q&A] [answered]
## Description

![image](https://user-images.githubusercontent.com/10399087/124757775-85d33d00-df60-11eb-9e9b-696f9758b29c.png)

### Steps to reproduce

Please check the screenshot

### Expected result

The images should be limited within the borders of a article

### Actual result

The images go beyond the boundary of the article

### Environment

{
  'doks-child-theme': '0.3.3',
  npm: '6.14.9',
  ares: '1.16.1',
  brotli: '1.0.9',
  cldr: '37.0',
  icu: '67.1',
  llhttp: '2.1.3',
  modules: '83',
  napi: '7',
  nghttp2: '1.41.0',
  node: '14.15.3',
  openssl: '1.1.1g',
  tz: '2020a',
  unicode: '13.0',
  uv: '1.40.0',
  v8: '8.4.371.19-node.17',
  zlib: '1.2.11'
}

**Comment by h-enk** [comment, marked as answer]
Hello @neotan, use the [image shortcode](https://getdoks.org/docs/recipes/images/#add-a-large-image) (or the `img-fluid` class — see [Responsive images](https://getbootstrap.com/docs/5.0/content/images/#responsive-images)).

**Reply by neotan** [reply]
@h-enk - Thanks for reply!
